README.md

```diff
@@ -2,16 +2,19 @@
 
 
 License : [![License](https://img.shields.io/badge/license-LGPL2-blue.svg)](https://github.com/analogdevicesinc/libiio/blob/master/LICENSE)
-Latest Stable Release : [![GitHub release](https://img.shields.io/github/release/analogdevicesinc/libiio.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
-Number of downloads :  [![Github All Releases](https://img.shields.io/github/downloads/analogdevicesinc/libiio/total.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
+Latest Release : [![GitHub release](https://img.shields.io/github/release/analogdevicesinc/libiio.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
+Downloads :  [![Github All Releases](https://img.shields.io/github/downloads/analogdevicesinc/libiio/total.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
+
+Forks :  [![GitHub forks](https://img.shields.io/github/forks/analogdevicesinc/libiio.svg?style=social&label=Fork&style=plastic)](https://github.com/analogdevicesinc/libiio/network)
+Watchers : [![GitHub watchers](https://img.shields.io/github/watchers/analogdevicesinc/libiio.svg?style=social&label=Watch&style=plastic)](https://help.github.com/articles/watching-repositories/)
 
 Scans : [![Coverity Scan Build Status](https://img.shields.io/coverity/scan/4796.svg)](https://scan.coverity.com/projects/analogdevicesinc-libiio)
 
 | Operating System        | Status       | Latest Binary |
 | ----------------------- |:------------:|:-------------:|
-| Windows                 | [![Windows Status](https://ci.appveyor.com/api/projects/status/github/analogdevicesinc/libiio?svg=true)](https://ci.appveyor.com/project/analogdevicesinc/libiio/branch/master) | [![Latest Windows build](https://ci.appveyor.com/api/projects/status/github/analogdevicesinc/libiio?svg=true)](https://ci.appveyor.com/project/analogdevicesinc/libiio/branch/master/artifacts)
+| Windows                 | [![Windows Status](https://ci.appveyor.com/api/projects/status/github/analogdevicesinc/libiio?svg=true)](https://ci.appveyor.com/project/analogdevicesinc/libiio/branch/master) | [![Latest Windows build](http://files.softicons.com/download/system-icons/box-icons-by-bjorn-lindberg/png/48x48/windows%20box.png)](https://ci.appveyor.com/api/projects/analogdevicesinc/libiio/artifacts/libiio-setup.exe?branch=master)
```

Comment:
So instead of pointing to a PNG of an external website I'd prefer host that PNG on Github too. I hosted them here:
https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/win_box.png
https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/deb_box.svg
https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/rpm_box.png

Are those free to use (creative commons)?